<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chessboard</class>
 <widget class="QMainWindow" name="Chessboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>933</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>933</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>933</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Chessboard</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>933</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="action_new"/>
    <addaction name="action_close"/>
   </widget>
   <widget class="QMenu" name="menu_about">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <widget class="QMenu" name="menu_mode">
    <property name="title">
     <string>模式</string>
    </property>
    <addaction name="action_man_machine"/>
    <addaction name="action_man_man"/>
   </widget>
   <widget class="QMenu" name="menu_option">
    <property name="title">
     <string>选项</string>
    </property>
    <addaction name="action_blackfirst"/>
    <addaction name="action_whitefirst"/>
   </widget>
   <widget class="QMenu" name="menu_difficulty">
    <property name="title">
     <string>难度</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_mode"/>
   <addaction name="menu_option"/>
   <addaction name="menu_difficulty"/>
   <addaction name="menu_about"/>
  </widget>
  <action name="action_close">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action_new">
   <property name="text">
    <string>开始新游戏</string>
   </property>
  </action>
  <action name="action_man_machine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>人机对战</string>
   </property>
  </action>
  <action name="action_man_man">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>人人对战</string>
   </property>
  </action>
  <action name="action_blackfirst">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>黑方先手</string>
   </property>
  </action>
  <action name="action_whitefirst">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>白方先手</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>action_close</sender>
   <signal>triggered()</signal>
   <receiver>Chessboard</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
